test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -138,4 +138,40 @@ using ChainRules: level2partition, level3partition, chol_blocked_rev, chol_unblo
             @test chol_unblocked_rev(B̄, B, true) ≈ chol_blocked_rev(B̄, B, 10, true)
         end
     end
+    @testset "qr" begin
+        @testset "the thing" begin
```

Comment:
Please explain

Reply:
I just reused the same structure as the test for cholesky. I can give it a more explicit name.

Reply:
Please do.